platform-core-module/src/main/java/com/softicar/platform/core/module/page/service/login/PageServiceLoginDiv.java

```diff
@@ -168,7 +168,7 @@ private void appendMaintenanceInProgressInfo(AGMaintenanceWindow maintenanceWind
 						DomMessageType.WARNING,
 						CoreI18n.MAINTENANCE_IS_CURRENTLY_IN_PROGRESS
 							.concat("\n")
-							.concat(CoreI18n.EXPECTED_END_ARG1.toDisplay(maintenanceWindow.getExpectedEnd().toHumanString()))));
+							.concat(CoreI18n.EXPECTED_END_ARG1.toDisplay(maintenanceWindow.getExpectedEnd().toDisplay()))));
```

Comment:
I think the `toDisplay` call should not be necessary, and I assume many developer will even forget about it an the locale will not be effective -> defect

Reply:
see the thread on `AGProgramExecutionTable`. closing this one.

Reply:
- still, I think `toDisplay()` must not be necessary
- see `I18nFormatArgumentsLocalizer` how we do that for numbers, also see comment below